with-utf8.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.18
 
--- This file has been generated from package.yaml by hpack version 0.34.5.
+-- This file has been generated from package.yaml by hpack version 0.35.2.
```

Comment:
I think we should open a follow-up PR that will add CI jobs which will check that cabal files are up to date